docs/cubeviz/plugins.rst

```diff
@@ -65,23 +69,23 @@ also saved to a dataset with the label specified in the :guilabel:`Model Label`
 
 Contours
 ========
+.. image:: img/contours_activated.png
 
-:guilabel:`Contours` can be accessed in the :guilabel:`Layer` tab in the viewer settings window.
-To activate :guilabel:`Contours`, click on the :guilabel:`Eye with a cross` icon and choose either the :guilabel:`Linear` icon for auto-contours or the :guilabel:`Custom` icon to set your own levels.
-
-.. image:: img/contour_location.png
+:guilabel:`Contours` can be created in the :guilabel:`Layer` tab in the viewer settings window.
+To activate :guilabel:`Contours`, click on the :guilabel:`Eye with a cross` icon and choose either
+the :guilabel:`Linear` icon for auto-contours or the
+:guilabel:`Custom` icon to set your own levels.
 
-.. image:: img/contours_activated.png
 
 The :guilabel:`Contours` of a 2D image can also be plotted over an image viewer.
-A user can create a 2D image using a plugin like :ref:`Collapse <collapse>` and then add it to an image viewer as a 2nd layer.
+A user can create a 2D image using a plugin like :ref:`Collapse <collapse>` and then add it
+to an image viewer as a 2nd layer. To visualize the contours of this 2D image over the image viewer,
+first go to the :guilabel:`Layer` tab, select the 2nd layer (which should be the 2D image), and
```

Comment:
First should be "add the data to the data drop down tab", then "go to the layer tab and select the 2nd layer." Although, the image already in the docs shows the second data being selected, so I will leave it up to your discretion whether or not to add this.

Reply:
I have made this change, but also rearranged the text a bit to say that contours of any second image can be overplotted, and that the Collapse image contours in the figure are one particular example of this.